modules/aggregation-layers/src/index.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015 - 2017 Uber Technologies, Inc.
```

Comment:
nit: do we need to update the year range to 2019?

Reply:
Will do in follow up PR.